### ISSUE TITLE
a11y: Accent color not AA-accessible according to WCAG 2.1 in dark mode

### ISSUE THREAD

**Issue by lagfarello** [tutao/tutanota#7680]
- [x] This is not a feature request (existing functionality does not work, **not** missing functionality).
 I will request features on [forum](https://www.reddit.com/r/tutanota/) or via support.
- [x] I've searched and did not find a similar issue.

**Describe the bug**
Accent color is not accessible according to WCAG 2.1 in dark mode (minimum contrast ration 4.5:1, https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html). This makes the text very difficult to read. Color customization is restricted to the "Whitelable feature".

**Screenshots**
![tuta-colors](https://github.com/user-attachments/assets/25404944-160e-4b26-8e1c-27a5dd8295dd)
![tuta-a11y](https://github.com/user-attachments/assets/e746ff4a-38db-43e0-9356-742b4a3656de)

**Test Notes**
- [ ] Make sure the dark red theme uses the color: #E99497
- [ ] Make sure the dark blue theme uses the color: #ACC7FF
- [ ] Make sure dark red theme has a contrast score of "AA" or above
- [ ] Make sure dark blue theme has a contrast score of "AA" or above

**Comment by charlag** [comment]
Thank you for reporting it, we already noticed it and we are going to fix it soon!

**Comment by andrehgdias** [comment]
Contrasts seems to be correct all across the app except for the little badge-counter used in the news icon and hamburger menu.
